app_legacy/Helpers/database/player-achievement.php

```diff
@@ -292,21 +289,25 @@ function getUsersRecentAwardedForGames(string $user, $gameIDsCSV, $numAchievemen
     // TODO: because of the "ORDER BY HardcoreAchieved", this query only returns non-hardcore
     //       unlocks if the user has more than $limit unlocks. Note that $limit appears to be
     //       default (5000) for all use cases except API_GetUserSummary
-    $query = "SELECT ach.ID, ach.GameID, gd.Title AS GameTitle, ach.Title, ach.Description, ach.Points, ach.BadgeName, (!ISNULL(aw.User)) AS IsAwarded, aw.Date AS DateAwarded, (aw.HardcoreMode) AS HardcoreAchieved
+    $query = "SELECT ach.ID, ach.GameID, gd.Title AS GameTitle, ach.Title, ach.Description, ach.Points, ach.BadgeName,
+                     aw.User AS IsAwarded, aw.Date AS DateAwarded, aw.HardcoreMode AS HardcoreAchieved
```

Comment:
sqllite didn't like the `!ISNULL` here, so I ended up fetching the column and doing the null check when fetching the rows.

Reply:
Both mysql and sqlite accept `NOT NULL` as far as i know